org.lflang/src/org/lflang/generator/EclipseErrorReporter.java

```diff
@@ -153,6 +146,21 @@ private String report(String message, Severity severity, Integer line, Path file
         return header + ": " + message;
     }
 
+    private IResource bestEffortGetIResource(Path path) {
+        IResource iResource = null;
+        if (path != null) {
+            iResource = FileUtil.getIResource(path);
+            if (iResource == null) {
+                // This operation is risky and may fail with a IllegalStateException or NPE.
```

Comment:
Shouldn't this be guarded by a try...catch then?  It's really bad when error reporting throws an exception because this results in hanging threads and useless messages about us caught exceptions.

Reply:
Now any `IllegalStateException` will fail silently. Our existing implementation assumes the workspace root will be non-null, so hopefully that will be safe.